crates/control/src/penalty_shot_direction_estimation.rs

```diff
@@ -52,10 +58,21 @@ impl PenaltyShotDirectionEstimation {
             }
             (PrimaryState::Playing, GamePhase::PenaltyShootout { .. }) => {
                 if let PenaltyShotDirection::NotMoving = self.last_shot_direction {
-                    if (context.ball_position.position.x
-                        - context.field_dimensions.penalty_marker_distance)
-                        .abs()
-                        > *context.moving_distance_threshold
+                    let is_ball_position_exceeed_moving_distance_threshold =
```

Comment:
No neeeeeed 😉

```suggestion
                    let is_ball_position_exceeding_moving_distance_threshold =
```

Reply:
Will fix it :P 
I split to parameters as there were multiple `&&` parts making it a bit hard to read.

Reply:
Please do not blindly apply suggestions on GitHub. This variable has now a different name but all remaining code uses still the old name... The CI is red. You also violated our policy that the author of a comment resolves it but you resolved this automatically by applying the suggestion. This comment needs more manual work. :wink: